test/package.json

```diff
@@ -9,19 +9,26 @@
     "coverage-report": "nyc report"
   },
   "dependencies": {
+    "@digitalbazaar/http-client": "^1.0.0",
+    "bcrypt": "^5.0.0",
     "bedrock": "^3.1.1",
-    "bedrock-account": "^4.0.0",
-    "bedrock-authn-token": "^2.2.0",
+    "bedrock-account": "github:digitalbazaar/bedrock-account#unique-email-field",
```

Comment:
has a release been made of `bedrock-account` with this feature?

Reply:
nope, a release has not been made yet. I'm testing the current work on top level modules and making sure all these changes are tested where these modules are being used.

Reply:
You'll, probably circling back around to this, but noting this need updating.

Reply:
Updated here cc8f8e0